src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.scss

```diff
@@ -1,16 +0,0 @@
-.exercise-scores-filter-toolbar {
-    /*
-      Window width required for same-line display of filters and buttons by site language:
-        - English: 1400px
-        - German: 1870px
-     */
-    @media (min-width: 1400px) {
```

Comment:
I don't think this is a good solution to hard-code it like this. What if a user zooms in or has a another font size set in his browser?